Doc/library/unittest.mock.rst

```diff
@@ -1369,7 +1369,8 @@ patch
     "as"; very useful if :func:`patch` is creating a mock object for you.
 
     :func:`patch` takes arbitrary keyword arguments. These will be passed to
-    the :class:`Mock` (or *new_callable*) on construction.
+    to construct a :class:`AsyncMock` if the patched object is an async function,
```

Comment:
double "to" now? 
a AsyncMock -> an AsyncMock.
I'm not sure this new wording is clearer.